packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -190,9 +190,9 @@ class ExpansionTile extends StatefulWidget {
   /// [children], and the `crossAxisAlignment` parameter is passed directly into the [Column].
   ///
   /// Modifying this property controls the cross axis alignment of each child
-  /// within its [Column]. Note that the width of the [Column] that houses
-  /// [children] will be the same as the widest child widget in [children]. It is
-  /// not necessarily the width of [Column] is equal to the width of expanded tile.
+  /// within its [Column]. The width of the [Column] that houses [children] will
+  /// be the same as the widest child widget in [children]. It is not
+  /// necessarily the width of [Column] is equal to the width of expanded tile.
```

Comment:
Consider something like:

```suggestion
  /// be the same as the widest child widget in [children]. The width of
  /// [Column] might not be equal to the width of the expanded tile.
```